dolphinscheduler-dao/src/main/resources/datasource.properties

```diff
@@ -21,6 +21,13 @@ spring.datasource.url=jdbc:postgresql://localhost:5432/dolphinscheduler
 spring.datasource.username=test
 spring.datasource.password=test
 
+# mysql
+#spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
```

Comment:
Require `spring.datasource.driver-class-name=com.mysql.jdbc.Driver` in `mysql-connector-java 5`

Reply:
Thx

Reply:
> Require `spring.datasource.driver-class-name=com.mysql.jdbc.Driver` in `mysql-connector-java 5`

I have changed this config